webpack.dev.js

```diff
@@ -25,12 +25,14 @@ module.exports = merge(common, {
   devServer: {
     index: '',
     proxy: {
-      '**': 'http://localhost:8000',
+      '**': 'http://backend:8000',
       '/ws': {
-        target: 'http://localhost:8000',
+        target: 'http://backend:8000',
         ws: true,
       },
     },
+    host: '0.0.0.0',
```

Comment:
This will break non-Docker development. Maybe we can if-branch this setting based on an env var or something.